components/app/leaderboard/LeaderboardProposal.vue

```diff
@@ -60,45 +60,47 @@
             </div>
 
             <!-- tags -->
-            <div>
+            <div class="flex flex-wrap">
```

Comment:
question(non-blocking): can we loose this wrapper and make the proposal-tags component a flex box?